src/test/app/Offer_test.cpp

```diff
@@ -1066,7 +1168,7 @@ class Offer_test : public beast::unit_test::suite
     }
 
     void
-    testcaseOfferCancelPastAndFuture()
+    testOfferCancelPastAndFuture()
```

Comment:
No harm, but I don't think the `env.close()` calls at the end of this test have any influence on the outcome.